.github/workflows/pull_request.yml

```diff
@@ -30,6 +30,30 @@ jobs:
       windows_build_command: docker\test-windows.ps1
       enable_windows_docker: false
 
+  full_test_run:
+    name: Full Test Run
+    if: contains(github.event.pull_request.labels.*.name, 'full-test-run')
+    uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
+    with:
+      # Linux
+      linux_exclude_swift_versions: '[{"swift_version": "nightly-6.0"},{"swift_version": "nightly-main"}]'
+      linux_env_vars: |
+        NODE_VERSION=v18.19.0
+        NODE_PATH=/usr/local/nvm/versions/node/v18.19.0/bin
+        NVM_DIR=/usr/local/nvm
+        CI=1
```

Comment:
is the main different that not setting FAST_TEST_RUN? If so I think we could reuse the job above and do something along lines of
FAST_TEST_RUN=${{ contains(github.event.pull_request.labels.*.name, 'full-test-run') && '0' || '1'}}

See https://github.com/actions/runner/issues/409#issuecomment-752775072

Reply:
Excellent idea